tldw_Server_API/tests/Slides/test_slides_api.py

```
@@ -671,6 +671,147 @@ def test_slides_templates_list_and_get(slides_client, tmp_path, monkeypatch):
+                "theme": "white",
+                "custom_css": {"selector": ".reveal"},
+            },
+            "fallback_policy": {"mode": "outline"},
+        },
+    )
+    assert create_resp.status_code == 422
+    assert create_resp.json()["detail"] == "invalid_visual_style_custom_css"
+
+
+def test_slides_create_rejects_resolved_style_with_non_string_custom_css(slides_client, monkeypatch):
+    def _broken_style(_self, style_id: str):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_broken_style`

(ANN202)